src/components/card/TailwindCard.tsx

```diff
@@ -0,0 +1,209 @@
+    cursor: pointer;
+  }
+  width: 98%;
+  @media (min-width: 1024px) {
+    margin-top: 0;
+    margin-bottom: 0;
+    width: 21rem;
+    height: 11.5rem;
+  }
+  @media screen and (max-width: 500px) {
+    width: 23rem;
+  }
+`;
+
+const TopContent = styled.div`
+  display: flex;
+  justify-content: space-between;
+  @media (min-width: 640px) {
+    justify-content: space-between;
+    gap: 1rem;
+  }
+`;
+
+const Link = styled.a`
+  font-weight: 700;
+  color: #111827;
+  @media (min-width: 1024px) {
+    font-size: 1.25rem;
+  }
+`;
+
+const Type = styled.p`
+  margin-top: 0.25rem;
+  font-size: 0.75rem;
+  line-height: 1rem;
+  font-weight: 700;
+  color: #4b5563;
+`;
+
+const ImageDiv = styled.div`
+  display: block;
+  margin-top: 0.5rem;
+  margin-bottom: 0.5rem;
+
+  @media (min-width: 640px) {
+    shrink: 0;
+  }
+
+  @media (min-width: 1024px) {
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+`;
+
+const Image = styled.img`
+  object-fit: cover;
+  border-radius: 9999px;
+  width: 4rem;
+  height: 4rem;
+  box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.05);
+`;
+
+const TagsDiv = styled.div`
+  display: flex;
+  column-gap: 0.75rem;
+  font-size: 0.875rem;
+  line-height: 1.25rem;
+  color: #6b7280;
+  max-width: 40ch;
+`;
+
+const Tags = styled.div`
+  display: inline-flex;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  padding-left: 8px;
+  padding-right: 8px;
+  align-items: center;
+  border-radius: 9999px;
+  font-size: 0.75rem;
+  line-height: 1rem;
+  font-weight: 700;
+  color: #047857;
+  text-transform: uppercase;
+  background-color: #a7f3d0;
+`;
+
+const LowerDiv = styled.dl`
+  display: flex;
+  margin-top: 1rem;
+  gap: 1rem;
+  width: 75%;
+  justify-content: space-between;
+
+  @media (min-width: 640px) {
+    gap: 1.5rem;
+  }
+`;
+
+const LowerItem = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const DescType = styled.dt`
+  font-size: 0.875rem;
+  line-height: 1.25rem;
+  font-weight: 500;
+  color: #4b5563;
+`;
+
+const DetailDesc = styled.dd`
+  font-size: 0.75rem;
+  line-height: 1rem;
+  text-align: center;
+  margin-left: 0;
+  color: #6b7280;
+`;
+
+export const TailwindCard = ({
+  name,
+  image,
+  type,
+  subject,
+  publisher,
+  tags,
+  styles,
+}: TailwindCardProps) => {
+  return (
+    <Container style={styles?.container}>
+      <TopContent style={styles?.headingDiv}>
+        <div>
+          <Link style={styles?.heading}>{name}</Link>
+          <Type style={styles?.type}>{type}</Type>
+        </div>
+        <ImageDiv style={styles?.imageDiv}>
+          <Image
+            style={styles?.image}
+            src={
+              image
+                ? image
+                : 'https://static.vecteezy.com/system/resources/previews/004/141/669/original/no-photo-or-blank-image-icon-loading-images-or-missing-image-mark-image-not-available-or-image-coming-soon-sign-simple-nature-silhouette-in-frame-isolated-illustration-vector.jpg'
```

Comment:
why this static URL you use?

Reply:
Sir, basically as many course not have images so, not to show blank thing, I have use the placeholder image